site-pages/login/login.css

```diff
@@ -0,0 +1,40 @@
+.login-main{
+    display: flex;
+    margin-top: 30px;
+    align-items: center;
+    justify-content: center;
+}
+.form-container{
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    align-self: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 20px;
+}
+.pass-rem{
+    display: flex;
+    justify-content: space-between;
+}
+.button{
+    background-color: transparent;
+}
+.label-inp{
+
+    margin-bottom: 1px;
+}
+.login-head{
+    font-weight: 500;
+    color: var(--primary-color);
+
+}
```

Comment:
Saw these classes while developing the signup css file https://github.com/Jia-A/dot-store/pull/6/files so if both of these css files(login.css and signup.css) are similar then you can create a common css file named as auth.css and store these common styles within it and keep the remaining things in these two files

Reply:
Okay, will do that definitely.